### PR TITLE
Feat: push Alpha to GCR

### PR DESCRIPTION
**Pull request by MrPink**

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -87,17 +86,24 @@ docker-compose run --rm gather-assets build
...
+    GCR_REPO_URL="https://eu.gcr.io"
+    IMAGE_REPO="eu.gcr.io/development-223016"
 
-if [ "$TRAVIS_BRANCH" == 'develop' ]; then
+    docker login -u _json_key -p "$(cat gcs_key.json)" $GCR_REPO_URL
+    docker_push ${VERSION} ${IMAGE_REPO}
     openssl aes-256-cbc -K $encrypted_422343ef1cd5_key -iv $encrypted_422343ef1cd5_iv -in gcs_key.json.enc -out gcs_key.json -d
     push-app-version --project gather-alpha --version $TRAVIS_COMMIT
-fi
+else
```
Comment (obdulia-losantos) [inline scripts/release.sh:109]:
```suggestion
fi
```

The alpha images are also pushed to Docker hub (but not the commit hash)

File: scripts/release.sh [hunk 2]
```diff
@@ -87,17 +86,25 @@ docker-compose run --rm gather-assets build
...
-docker logout
+    GCR_REPO_URL="https://eu.gcr.io"
+    IMAGE_REPO="eu.gcr.io/development-223016"
 
-if [ "$TRAVIS_BRANCH" == 'develop' ]; then
+    docker login -u _json_key -p "$(cat gcs_key.json)" $GCR_REPO_URL
+    docker_push ${VERSION} ${IMAGE_REPO}
     openssl aes-256-cbc -K $encrypted_422343ef1cd5_key -iv $encrypted_422343ef1cd5_iv -in gcs_key.json.enc -out gcs_key.json -d
     push-app-version --project gather-alpha --version $TRAVIS_COMMIT
+else
```
Comment (obdulia-losantos) [inline scripts/release.sh:109]: could this be an extra? I mean we still need to push `alpha` to docker hub :grimacing:

**Review by obdulia-losantos — APPROVED** [review]

File: scripts/release.sh (hunk 2 above)
Comment (MrPink) [reply scripts/release.sh:109]: just added it, let me know if you want it a different way

Comment (obdulia-losantos) [reply scripts/release.sh:109]: I will revisit this script some day in the future... but so far is good enough.

**Pull request merged**